src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestStruct.cs

```diff
@@ -28,6 +28,10 @@ public class SimpleTestStruct : ITestClass
         public DateTime MyDateTime { get; set; }
         public DateTimeOffset MyDateTimeOffset { get; set; }
         public SampleEnum MyEnum { get; set; }
+        public SampleInt64Enum MyInt64Enum { get; set; }
+        public SampleUInt64Enum MyUInt64Enum { get; set; }
+        public SampleStruct MySampleStruct { get; set; }
+        public SimpleTestClass MySampleTestClass { get; set; }
```

Comment:
nit: Should the name be `MySimpleTesClass` (and same with the other usages of `Sample`)?

```suggestion
        public SimpleTestClass MySimpleTestClass { get; set; }
```